### PR TITLE
Feature/oe 182 reports commision tracker

### PR DESCRIPTION
**Pull request by Ron-Junior**
## Goal

Page with commission tracker

## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: tests/Feature/Livewire/ReportsOverrideTest.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    }
+
+     /** @test */
+     public function it_should_calculate_total_commission()
+     {
+
+         Livewire::test(ReportsOverview::class)
+            ->set('departmentId', $this->department->id)
+            ->assertSee($this->customer->first_name);
+     }
```
[the remark this answers is not in the document]
Comment (Ron-Junior) [reply tests/Feature/Livewire/ReportsOverrideTest.php]: I don't know why, but in the project, this code doesn't exist.
